docs/compressing-images.md

```diff
@@ -0,0 +1,46 @@
+# TEAMMATES Compressing Images
+
+This document describes the steps to compress an image to be used in the TEAMMATES website.
```

Comment:
> images inside the app

Just the website, since compressing is used to improve page load speeds for production.